internal/api/tests/private/runsCancelV2_test.go

```diff
@@ -0,0 +1,153 @@
+package private
+
+import (
+	"net/http"
+	"playbook-dispatcher/internal/api/controllers/public"
+	dbModel "playbook-dispatcher/internal/common/model/db"
+	"playbook-dispatcher/internal/common/utils/test"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func cancelV2(payload *ApiInternalV2RunsCancelJSONRequestBody) (*RunsCanceled, *ApiInternalV2RunsCancelResponse) {
+	resp, err := client.ApiInternalV2RunsCancel(test.TestContext(), *payload)
+	Expect(err).ToNot(HaveOccurred())
+	res, err := ParseApiInternalV2RunsCancelResponse(resp)
+	Expect(err).ToNot(HaveOccurred())
+	Expect(res.StatusCode()).To(Equal(http.StatusMultiStatus))
+
+	return res.JSON207, res
+}
+
+func minimalV2Cancel() CancelInputV2 {
+	return CancelInputV2{
+		OrgId:     OrgId("5318290"),
+		Principal: Principal("test_user"),
+		RunId:     public.RunId("eb05b1a7-6613-442f-9ca6-1d7f83188915"),
+	}
+}
+
+var _ = Describe("runsCancel V2", func() {
+	db := test.WithDatabase()
+
+	It("sends 400 for invalid orgId", func() {
+		payload := minimalV2Cancel()
+		payload.OrgId = "1234"
+
+		runs, _ := cancelV2(&ApiInternalV2RunsCancelJSONRequestBody{payload})
+		Expect((*runs)[0].Code).To(Equal(400))
+	})
+
+	It("creates cancelation message on successful operation", func() {
+		satId, _ := uuid.Parse("95cbea43-bb85-4153-96c2-eb2474b3e2b3")
```

Comment:
```suggestion
		satId := uuid.MustParse("95cbea43-bb85-4153-96c2-eb2474b3e2b3")
```

in cases where you want to ignore the error (tests) it's better to use the MustParse variant, which panics should there be an error after all